api-service/api-server/src/setup/custom_uploaders/upload_toxicity_subtypes.ts

```diff
@@ -49,10 +49,10 @@ import * as readline from 'readline';
 import * as seedrandom from 'seedrandom';
 import * as yargs from 'yargs';
 
-import * as crowdsourcedb from '../cs_db';
-import * as db_types from '../db_types';
+import * as crowdsourcedb from '../../cs_db';
```

Comment:
I think you'll want to change the comments at https://github.com/conversationai/conversationai-crowdsource/pull/22/files#diff-af03a2c660d78426b210a262d95c6f00R20 with the new script location

Reply:
Done